third_party/qemu/README.md

```diff
@@ -1,48 +1,112 @@
 # QEMU
 
-By default, the build system will download a pre-built release of our [QEMU fork](https://github.com/lowRISC/qemu/).
-In order to support local development and testing, the build system also supports a way to build QEMU from source.
-See instructions below.
+For instructions on setting up QEMU for local development and troubleshooting steps, see the [setup guide](./setup.md).
 
-## Building from source using Bazel
+## Introduction
 
-To perform local development, you first need to check out a copy of [QEMU fork](https://github.com/lowRISC/qemu/) and switch to the correct branch.
+[QEMU](https://www.qemu.org/) is an open-source project which primarily aims to support virtualization, user-mode emulation and full-system (machine) emulation flows.
+OpenTitan is an example of full-system emulation, where we emulate an OpenTitan RISC-V 32-bit machine (equivalent to a "top") to support running OpenTitan programs on any generic supported host machine, such as an x86-64 PC.
 
-The following step must only be done once:
-```bash
-# Run the following commands at the root of your QEMU checkout.
-touch REPO.bazel
-ln -s "/path/to/your/opentitan/repo/third_party/qemu/BUILD.qemu_opentitan.bazel" "BUILD.bazel"
-```
+QEMU's predominant advantage is its speed: internally, QEMU uses a dynamic translation backend called [TCG](https://www.qemu.org/docs/master/devel/tcg.html) (_Tiny Code Generator_) which optimizes translation of blocks of guest (e.g. RISC-V) instructions to equivalent host (e.g. x86-64) instructions to execute.
```

Comment:
The phrasing seems a bit off - the dynamic translator doesn't optimise the translation of blocks, it translates blocks of guest code into optimised native code.

Also a minor nit: I think "The predominant/primary advantage of QEMU is its speed" reads better.